create-index.md

```diff
@@ -1,18 +1,31 @@
 ---
 title: CREATE INDEX
-toc: true
+toc: false
 ---
 
-## Description
+The `CREATE INDEX` [statement](sql-statements.html) creates an index for a table. Indexes are used to quickly locate data without having to look through every row of a table. 
+
+{{site.data.alerts.callout_info}}Indexes are automatically created for the primary key of a table and any columns with a unique constraint.{{site.data.alerts.end}}
+
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/create_index.html %}
 
-## Privileges
+## Required Privileges
+
+The user must have the `CREATE` privilege on the table.
```

Comment:
the `ALTER` documentation links to privilege.htm. I think we should either do that everywhere, or nowhere, but we should be consistent.